src/styles/index.css

```diff
@@ -1,3 +1,2 @@
 @import 'vars.css';
 @import 'base.css';
-@import 'utils.css';
```

Comment:
I've merged the content of `utils.css` into `base.css`. We had reflex overrides in there that weren't `utils` anyway, and in the dark mode query, I use a couple of utility classes, so it doesn't seem worth it to try to have two separate files.